lib/v2/index.d.ts

```diff
@@ -1,12 +1,16 @@
 import Vue from 'vue'
 declare const isVue2: boolean
 declare const isVue3: boolean
+declare const Vue2: Vue
 declare const version: string
 declare const install: (vue?: Vue) => void
+/** @deprecated */
```

Comment:
```suggestion
/** 
 * @deprecated To avoid bringing in all the tree-shakable modules, this API has been deprecated. Use `Vue2` or named exports instead.
 * Refer to https://github.com/vueuse/vue-demi/issues/41
 */
```